example/string_generator.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+"""
+CherryPy String Generator Service.
+
+This is the example random string generator service, slightly
+modified for swagger to not use session support.
+"""
+import random
+import string
+import os
+import cherrypy
+
+STRING_FILE = 'string.txt'
```

Comment:
Does it have to be a real file in the FS? I'd say you could use in-memory file instead.
What do you think about this suggestion?